fix apply global selectize options to arraySelectize

mneofit opened the pull request:
While using selectize, overrided "selectize_options" were not applied to selectize instance created by arraySelectize.
Fixed error.

pmk65 commented:
@mneofit Sorry it took so long. Do you have an example test case I can use to test you changes?

mneofit commented:
@pmk65 You can test my changes on json editor selectize integration example. I have moved it into separate fiddle: https://jsfiddle.net/0L7nk36f/
In lines 64 and 65 I have added some options to configure selectize, but it completely ignores in the final render.

germanbisurgi commented:
Looks good to me. A little test would be a nice to have.

schmunk42 commented:
@germanbisurgi How should the test look like?

germanbisurgi commented:
@schmunk42 a test pages that builds the editor from a schema. Check if the fields exist (quering elements paths and attributes) and control that the returned output (getValue) reflects the final goal

mneofit commented:
@germanbisurgi Test is added. While writing this test I have spotted that selectized array in the array is not receiving default values, fixed it also.

Here is the fiddle for what I mean https://jsfiddle.net/onwxekt2/
Note that although default values are set, they are not displaying in the array.

mneofit commented:
@germanbisurgi test is added. While writing this test I have spotted that selectized array in the array is not received default values, fixed it also.

Here is the fiddle for what I mean https://jsfiddle.net/onwxekt2/
Note that although default values are set, they are not displaying in the array.

schmunk42 commented:
Thank you!
